notebooks/triton/aks-gpu-deploy.ipynb

```diff
@@ -0,0 +1,268 @@
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Uncomment the below line to see the contents of setup.py\n",
+    "# %cat ../scripts/setup.py"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "\n",
+    "!python setup.py --compute_loc='westus2' --vm_size='Standard_NC6'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Register the model\n",
+    "Register an existing trained model, add description and tags.\n",
+    "\n",
+    "** Note: ** Under `model_path` there must be a sub-directory named `triton`, which has the structure of a Triton [Model Repository](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_repository.html#repository-layout)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from azureml.core.model import Model\n",
+    "import os\n",
+    "\n",
+    "model = Model.register(\n",
+    "    model_path=os.path.join(\"..\", \"..\", \"models\", \"triton\"),\n",
+    "    model_name=\"densenet_onnx\",\n",
+    "    tags={\"area\": \"Image classification\", \"type\": \"classification\"},\n",
+    "    description=\"Image classification trained on Imagenet Dataset\",\n",
+    "    workspace=ws,\n",
+    ")\n",
+    "\n",
+    "print(model.name, model.description, model.version)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy the model as a web service\n",
+    "First create a scoring script. You can see the one we created for you in the `scripts` directory. Then, create an InferenceConfig and a DeploymentConfig and call Model.deploy().\n",
+    "\n",
+    "Note that this step may take 10-15 minutes to run. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "\n",
+    "from azureml.core.webservice import AksWebservice\n",
+    "from azureml.core import Environment\n",
+    "from azureml.core.model import Model, InferenceConfig\n",
+    "\n",
+    "\n",
+    "aks_service_name = \"triton-densenet-onnx\"\n",
+    "env = Environment.get(ws, \"AzureML-Triton\").clone(\"My-Triton\")\n",
+    "\n",
+    "for pip_package in [\"pillow\"]:\n",
+    "    env.python.conda_dependencies.add_pip_package(pip_package)\n",
+    "\n",
+    "inference_config = InferenceConfig(\n",
+    "    # This entry script is where we dispatch a call to the Triton server\n",
+    "    entry_script=\"score_densenet.py\",\n",
+    "    source_directory=os.path.join(\"source_dir\"),\n",
+    "    environment=env,\n",
+    ")\n",
+    "\n",
+    "aks_config = AksWebservice.deploy_configuration(\n",
+    "    cpu_cores=1, memory_gb=4, gpu_cores=1, compute_target_name=\"aks-gpu-deploy\"\n",
```

Comment:
I thought the compute target was created already? regardless we can create one in the test workspace so we can avoid creating one everytime in the notebook